src/site/_data/site.js

```diff
@@ -32,8 +32,8 @@ module.exports = {
   subscribeForm:
     'https://services.google.com/fb/submissions/591768a1-61a6-4f16-8e3c-adf1661539da/',
   thumbnail: 'image/FNkVSAX8UDTTQWQkKftSgGe9clO2/uZ3hQS2EPrA9csOgkoXI.png',
-  isBannerEnabled: false,
-  banner: '',
+  isBannerEnabled: true,
+  banner: 'The PWA community is coming together for #PWASummit22. Have a great story about developing a web app? [Submit your talk today](https://sessionize.com/pwa-summit-2022). ',
```

Comment:
There's a linting error:

```
/home/runner/work/web.dev/web.dev/src/site/_data/site.js
  36:10  error  Insert `⏎···`  prettier/prettier
✖ 1 problem (1 error, 0 warnings)
```

Reply:
Thank you for checking Jeff! very weird I swear I fixed that issue cause otherwise it didn't let me push the branch... but who knows ¯\_(ツ)_/¯ Thanks again.